Sources/Classes/core/SecureStorage/MiniAppSecureStorage.swift

```diff
@@ -67,19 +67,22 @@ class MiniAppSecureStorage: MiniAppSecureStorageDelegate {
             return
         }
 
-        do {
-            try database.set(dict: dict)
-            try database.save(completion: { result in
-                switch result {
-                case .success:
-                    completion?(.success(true))
-                case let .failure(error):
-                    completion?(.failure(error))
-                }
-            })
-        } catch {
-            completion?(.failure(.storageIOError))
-            return
+        DispatchQueue.global(qos: .userInitiated).async {
```

Comment:
Shouldn't we do the same for Delete as well ?

Reply:
yeah true, fixed it